addons/website/static/tests/tours/homepage_edit_discard.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module alias=website.homepage_edit_discard*/
```

Comment:
Why do you need an alias?

Reply:
@Goaman
I have observed some test tours in which it's used. There is no particular reason for this. 
If it's not needed I will remove it.

Reply:
Indeed it is not needed.

Reply:
@Goaman Done!